### PR TITLE
changed placeholder to be consistent with labels

### LINKED ISSUE #6845: Inconsistency in field name and placeholder
### What is the expected behaviour?
Field name and the corresponding placeholder should be consistent

### What is the current behaviour?
Users tab: Enter full name (field name) vs Enter first name (placeholder)
Groups tab: Add new group (field name) vs Enter name (placeholder)

### How to reproduce the issue?

1. Go to workspace settings- Users tab and Groups tab
2. Verify the field name and the placeholders
3.

### Screenshots or Screencast

https://github.com/ToolJet/ToolJet/assets/125240229/8a87cc40-da74-40ed-9be3-595b7fa635ac

### PR DESCRIPTION
**Pull request by JunedKhan101**
fixes #6845 
Change enterName value in en.json just in case if it's used somewhere else.

### CODE REVIEW

File: frontend/src/ManageOrgUsers/InviteUsersForm.jsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ function InviteUsersForm({
                       <input
                         type="text"
                         className="form-control"
-                        placeholder={t('header.organization.menus.manageUsers.enterFirstName', 'Enter full name')}
+                        placeholder={t('header.organization.menus.manageUsers.enterFulltName', 'Enter full name')}
```
Comment (stepinfwd) [inline frontend/src/ManageOrgUsers/InviteUsersForm.jsx:91]: There is a typo here  and if `organization.menus.manageUsers.enterFullName` is not available add it in `en.json`

**Review by stepinfwd — CHANGES_REQUESTED** [review]

File: frontend/assets/translations/en.json [hunk 2]
```diff
@@ -267,7 +267,7 @@
                         "createNewGroup": "Create new group",
                         "updateGroup": "Update group",
                         "addNewGroup": "Add new group",
-                        "enterName": "Enter Name",
+                        "enterName": "Enter Group Name",
```
Comment (stepinfwd) [inline frontend/assets/translations/en.json:270]: change this to `Enter group name`

Comment (JunedKhan101) [reply frontend/assets/translations/en.json]: Implemented the change

**Comment by stepinfwd** [comment]
Hi @JunedKhan101 requested couple for changes

**Comment by JunedKhan101** [comment]
@stepinfwd What are the requested changes? I can't see them on my PR. I have resolved changes you mentioned earlier so if you are referring to that I am just letting you know that I have resolved and pushed my new changes.

**Review by stepinfwd — CHANGES_REQUESTED** [review]

File: frontend/src/ManageGroupPermissions/ManageGroupPermissions.jsx [hunk 3]
```diff
@@ -262,7 +262,7 @@ class ManageGroupPermissionsComponent extends React.Component {
                         className="form-control"
                         placeholder={this.props.t(
                           'header.organization.menus.manageGroups.permissions.enterName',
-                          'Enter Name'
+                          'Enter Group Name'
```
[the remark this answers is not in the document]
Comment (JunedKhan101) [reply frontend/src/ManageGroupPermissions/ManageGroupPermissions.jsx]: Resolved

File: frontend/assets/translations/en.json [hunk 4]
```diff
@@ -258,6 +258,7 @@
                     "enterFirstName": "Enter First Name",
                     "enterLastName": "Enter Last Name",
                     "enterEmail": "Enter email id",
+                    "enterFullName": "Enter full name",
                     "enterFulltName": "Enter full name",
```
[the remark this answers is not in the document]
Comment (JunedKhan101) [reply frontend/assets/translations/en.json]: Removed

**Comment by JunedKhan101** [comment]
@stepinfwd Resolved PR comments please verify.

**Comment by JunedKhan101** [comment]
@stepinfwd The PR is updated Please check.

**Review by stepinfwd — APPROVED** [review]

**Review by kavinvenkatachalam — APPROVED** [review]

**Pull request merged**